src/nupic/regions/SDRClassifierRegion.py

```diff
@@ -0,0 +1,500 @@
+#!/usr/bin/env python
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""
+This file implements the SDR Classifier region. See the comments in the class
+definition of SDRClassifierRegion for a description.
+"""
+
+import warnings
+
+from nupic.bindings.regions.PyRegion import PyRegion
+from nupic.algorithms.sdr_classifier_factory import SDRClassifierFactory
+from nupic.support.configuration import Configuration
+
+try:
```

Comment:
Once we have it working for Windows we can remove it altogether. Open to suggestions in the interim but let's not hold up PRs like this on it.